### PR TITLE
LOVE 1577- Added ACCEPT_EULA environment variable, and CFN parameter

### PR DESCRIPTION
**Pull request by spaliwal-xl**
We have set AcceptEula default value to none as per the following documentation:
[environment-variables.md](https://github.com/xebialabs/xl-docker-images/blob/master/docs/environment-variables.md)

This PR has two scenarios for testing:

1. **Quick start if you have a license:** (use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values in [taskcat.yml](https://github.com/xebialabs/quickstart-xebialabs-devops-platform/blob/LOVE-1577/ci/taskcat.yml) file)

         AcceptEula: none
         XLDeployLicense: BASE64
         XLReleaseLicense: BASE64

2. **Quick start if you do not have a license:** (use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values in [taskcat.yml](https://github.com/xebialabs/quickstart-xebialabs-devops-platform/blob/LOVE-1577/ci/taskcat.yml) file)

        AcceptEula: yes
        XLDeployLicense: trial
        XLReleaseLicense: trial

### CODE REVIEW

File: ci/taskcat.yml [hunk 1]
```diff
@@ -34,6 +34,12 @@ tests:
...
       LoadBalancerType: internet-facing
       InstallBastionHost: 'false'
+      # If you have a BASE64 license then use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values
+      AcceptEula: none
       XLDeployLicense: BASE64
       XLReleaseLicense: BASE64
+      # If you do not have a BASE64 license then use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values
+      # AcceptEula: yes
+      # XLDeployLicense: trial
+      # XLReleaseLicense: trial
```
Comment (sffortytwo) [inline ci/taskcat.yml:44]: Why are you committing a commented block?

Comment (spaliwal-xl) [reply ci/taskcat.yml:44]: This block is used only if the user does not have a BASE64 license. Please refer to scenario 2 

**Quick start if you do not have a license**: (use below AcceptEula, XLDeployLicense, and XLReleaseLicense parameter values in [taskcat.yml](https://github.com/xebialabs/quickstart-xebialabs-devops-platform/blob/LOVE-1577/ci/taskcat.yml) file)

 AcceptEula: yes
 XLDeployLicense: trial
 XLReleaseLicense: trial

If the user has BASE64 license then use scenario 1

**Quick start if you have a license**: (use below AcceptEula, XLDeployLicense and XLReleaseLicense parameter values in [taskcat.yml](https://github.com/xebialabs/quickstart-xebialabs-devops-platform/blob/LOVE-1577/ci/taskcat.yml) file)

  AcceptEula: none
  XLDeployLicense: BASE64
  XLReleaseLicense: BASE64

I have commented that block just for either or condition of license availability

Comment (spaliwal-xl) [reply ci/taskcat.yml:44]: Please hold on PR review as email communication is still going on for EULA approach in AWS Quickstart. Once it is finalized then I'll commit corrected files and let you know for the review.

**Comment by spaliwal-xl** [comment]
I am closing this PR as the current approach is not correct. A different approach is finalized in which AcceptEula = yes parameter is must in any condition. I'll raise a PR again with corrected files.

**Pull request closed without merging**